### ISSUE TITLE
[Change in C4IdC Video Player]: Change position of in-video call-to-actions on the video-player

### ISSUE THREAD

**Issue by tweenn** [carbon-design-system/carbon-for-ibm-dotcom#11625]
### Summary

Due to accessibility complaints, we want to change in-video call-to-actions position from bottom-left to top-right just like youtube's cards in order to get then away from overlapping with closed captions - and Gerrard Ringuette (@jerryr) asked me to get a UX Review with you guys if possible.
There is no code-action needed for this change, just the design review if possible.

### Relevant information

Our in-video call-to-actions [ctas] were designed originally by part of the brand team back in 2019 (by folks now at blue-studio or ibm.com) to appear in the bottom-left corner of the player as you can see in the screenshots below [as is]

As this positioning overlaps with closed captions - either because the closed-caption contain a big sentence or the user configured the player to display big letters - creating an accessibility violation, we want to change it to the top-right corner (just like youtube's cards) as in the screenshots below:

-------

### AS IS

![image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/1704162/dd16f682-9458-46de-b4a1-6281175f26a4)

-------

### Youtube Card Example

![image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/1704162/dd5f8f6a-6683-4898-8752-d42f784e2911)

-------

### TO BE

![image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/1704162/162f06ca-c4c4-4bd9-b771-2c2e06747bc9)

![image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/1704162/5168b9ce-3e66-4dda-82d8-6cf9fd7c69fe)

-------

Ps - As I get this a lot, the "is-this-helpful" in-screen ctas use both thumbs-up and thumbs-down @carbon/icons.

### Name and organization

Felipe Zuntini - IBM Mediacenter's Architect

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
